dev/com.ibm.ws.opentracing/resources/com/ibm/ws/opentracing/resources/Opentracing.nlsprops

```diff
@@ -86,6 +86,14 @@ OPENTRACING_TRACERFACTORY_RETURNED_NULL=CWMOT0006E: Invocation of user supplied
 OPENTRACING_TRACERFACTORY_RETURNED_NULL.explanation=The feature constructs Tracer implementations with a user provided OpentracingTracerFactory. The OpentracingTracerFactory.newInstance method returned null.
 OPENTRACING_TRACERFACTORY_RETURNED_NULL.useraction=Determine why the user supplied OpentracingTracerFactory newInstance method returned null.
 
+OPENTRACING_FILTER_PATTERN_BLANK=CWMOT0007E: The pattern attribute of an opentracing excludeSpans or includeSpans configuration element is blank.
+OPENTRACING_FILTER_PATTERN_BLANK.explanation=A valid URI pattern must be specified.
+OPENTRACING_FILTER_PATTERN_BLANK.useraction=Consult the documentation on valid patterns.
+
```

Comment:
**This set of messages look good! I have some minor suggestions for clarity and a question about the useraction message.**
+OPENTRACING_FILTER_PATTERN_BLANK.explanation=A valid URI pattern must be specified **and cannot be blank.**
+OPENTRACING_FILTER_PATTERN_BLANK.useraction=Consult the documentation on valid patterns. 
**Is there specific documentation about the valid patterns where users should look?**

Reply:
Thanks. Updated the explanation in the latest commit. The documentation about the valid patterns is right now just in the javadoc in the code: https://github.com/kgibm/open-liberty/blob/5b714fb2f5460e0121229ec0818856a98dce287a/dev/com.ibm.ws.opentracing/src/com/ibm/ws/opentracing/filters/SpanFilter.java#L16  That will need to be put into a KC documentation page. What's the process for that?